backend/migrations/README.md

```diff
@@ -1,24 +1,53 @@
 # Managing DB with Alembic locally
+In data.all we use [Alembic](https://alembic.sqlalchemy.org/en/latest/)  -- a lightweight database migration tool for usage with the SQLAlchemy Database Toolkit for Python.
+Alembic relies on the database's current state to generate and apply migrations accurately.
+Alembic determines the changes to be made by comparing the current state of the database with the desired state specified in your SQLAlchemy models. This process, known as schema diffing, requires an existing database to identify differences.
+Alembic generates migration scripts based on the detected differences between the current database schema and the desired schema defined in your application code. This generation is dependent on the actual structure and content of the database.
 
-When we run ```docker-compose up``` the postgres container is created with no tables or schemas.
+In order to create and test migrations locally you will have to create a local database.
+
```

Comment:
I think we are mixing 2 ways of running migrations. One is the `dkrcompose` environment that uses declarative base and the other is if we create a new schema maybe called "local". I would argue that both are useful, "local" allows users to play and break the schema as much as they want. "dkrcompose" allows them to do more complex testing (with populated data from a previous version)